sidebars.js

```diff
@@ -28,13 +36,7 @@ module.exports = {
       type: 'category',
       label: 'Platform Builder Guide',
       items: [
-        {
-          'Design Abstraction': [
-            'platform-engineers/overview',
-            'application',
-            'platform-engineers/definition-and-templates',
-          ]
-        },
+        'platform-engineers/definition-and-templates',
         {
```

Comment:
this file will automatically synced from https://github.com/oam-dev/kubevela/tree/master/docs